runtime/doc/autocmd.txt

```diff
@@ -994,7 +994,7 @@ expect.
 Environment variables can be used in a pattern: >
 	:autocmd BufRead $VIMRUNTIME/doc/*.txt  set expandtab
 And ~ can be used for the home directory (if $HOME is defined): >
-	:autocmd BufWritePost ~/.vimrc   so ~/.vimrc
+	:autocmd BufWritePost ~/.config/nvim/init.vim   so <afile>
```

Comment:
Regarding all these references to `~/.config/...`: shouldn't these instead say `$XDG_CONFIG_HOME/...`?